### ISSUE TITLE
(qemu) Older versions do not install due to changed download location

### ISSUE THREAD

**Issue by soul4soul** [AdmiringWorm/chocolatey-packages#173]
**To Reproduce**
```
PS C:\Windows\system32> choco upgrade -y --allow-downgrade qemu --version=2019.08.15
Chocolatey v0.10.15
Upgrading the following packages:
qemu
By upgrading you accept licenses for the packages.
Progress: Downloading Qemu 2019.08.15... 100%

Qemu v2019.08.15
qemu package files upgrade completed. Performing other installation steps.
Attempt to get headers for https://qemu.weilnetz.de/w64/qemu-w64-setup-20190815.exe failed.
  The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://qemu.weilnetz.de/w64/qemu-w64-setup-20190815.exe'. Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
Downloading Qemu 64 bit
  from 'https://qemu.weilnetz.de/w64/qemu-w64-setup-20190815.exe'
ERROR: The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://qemu.weilnetz.de/w64/qemu-w64-setup-20190815.exe'. Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
The upgrade of qemu was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\Qemu\tools\chocolateyInstall.ps1'.
 See log for details.

Chocolatey upgraded 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - qemu (exited 404) - Error while running 'C:\ProgramData\chocolatey\lib\Qemu\tools\chocolateyInstall.ps1'.
 See log for details.
```

**Desktop (please complete the following information):**

- OS:  Windows 10

**Additional context**
Mr. Weil moves installers for previous years into a subfolder when the year rolls over. This breaks the download link used by the QEMU chocolatey package.

https://qemu.weilnetz.de/w64/ => https://qemu.weilnetz.de/w64/2019/

Somewhat related, is it possible to allow the real version number of QEMU to be used for install purposes? IE 4.1.0 instead of 2019.08.15.

**Comment by AdmiringWorm** [comment]
Thank you for opening this issue (and sorry for not responding earlier).

I will add the necessary code to the package to use a fallback url in the case the normal url is no longer available. (When I initially created the package, the current version was only provided at the root directory, not in a sub directory).

Unfortunately this will only be available for versions going forward, and not for older versions.

> Somewhat related, is it possible to allow the real version number of QEMU to be used for install purposes? IE 4.1.0 instead of 2019.08.15.

Unfortunately I can not do that.
When I initially created the package, there was no easy way to determine the actual version of the application (short of launching the application and look at the version number).
Because I decided back then to use the date it was released, I can not go back to the actual version number as it would then not show up when a user runs `choco upgrade`. (Remember, the version `2020.02.01` is a higher version than `4.2.0`).
